docs/partiql-tests-schema-proposal.md

```diff
@@ -121,22 +121,23 @@ Tests whether a given PartiQL statement evaluates to the expected result. For no
 - [optional] input evaluation environment, `env` - struct
   - defaults to using environments specified in file (i.e. `envs`). If `envs` is unspecified, defaults to an empty
   environment with no bindings
-- [optional] evaluation `options` other than the defaults - struct
-    - e.g. typing mode - strict vs permissive (permissive may be most useful)
```

Comment:
This was originally a struct to follow all the different compile options specified in the Kotlin reference implementation (see [CompileOptions.kt](https://github.com/partiql/partiql-lang-kotlin/blob/main/lang/src/org/partiql/lang/eval/CompileOptions.kt)). After examining the Kotlin implementation's options and the specification, determined that we only really care about the "typing mode" (renamed to evaluation mode).